### PR TITLE
Add `llama download` support for multiple models with comma-separated list

### PR DESCRIPTION
**Pull request by ABucket**
Address issue #233 , allowing users to pass a comma-separated list of model IDs with the --model-ids argument and matching meta URLs via --meta-url.

### CODE REVIEW

**Review by yanxi0830 — APPROVED** [review]
Thanks! LG!

**Review by yanxi0830 — CHANGES_REQUESTED** [review]
Could you install pre-commit and lint / format the code?

File: llama_stack/cli/download.py [hunk 1]
```diff
@@ -148,31 +153,43 @@ def run_download_cmd(args: argparse.Namespace, parser: argparse.ArgumentParser):
...
         return
 
-    if args.model_id is None:
-        parser.error("Please provide a model id")
+    if args.model_ids:
+        model_ids = [model_id.strip() for model_id in args.model_ids.split(",")]
+    elif args.model_id:
+        model_ids = [args.model_id]
+    else:
+        parser.error("Please provide a model id or a list of model ids (--model-ids)")
```
Comment (ashwinb) [inline llama_stack/cli/download.py:161]: nit: `Please provide a model using --model-id or --model-ids`

File: llama_stack/cli/download.py [hunk 2]
```diff
@@ -48,6 +48,11 @@ def setup_download_parser(parser: argparse.ArgumentParser) -> None:
         required=False,
         help="See `llama model list` or `llama model list --show-all` for the list of available models",
```
Comment (ashwinb) [inline llama_stack/cli/download.py:49]: can we just make this `nargs='+'` so you can provide it multiple times like `--model-id A --model-id B`?

**Review by ashwinb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**